packages/smooth_app/lib/pages/page_manager.dart

```diff
@@ -34,6 +35,7 @@ class PageManagerState extends State<PageManager> {
   };
 
   BottomNavigationTab _currentPage = BottomNavigationTab.Scan;
+  List<Widget> _tabs = <Widget>[];
```

Comment:
Is there anything stopping you from directly adding all the _buildOffstageNavigator's up here

Reply:
To be frank, I don't have a MUST reason to put it here😄